todo_app/app.py

```diff
@@ -1,10 +1,13 @@
 from flask import Flask, render_template, request , redirect
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items, add_item
 
+from todo_app.data.trello_item import add_items, get_items, complete_item
+
+import os
 app = Flask(__name__)
 app.config.from_object(Config())
 
+print(os.getenv("TRELLO_API_KEY"))
```

Comment:
Probably best to remove this debugging line (as this will lead to the API key being leaked in various logs).